OSDOCS-5240 #57427: review rendered page with important fixes on the structure (#4)
maxwelldb/openshift-docs: modules/machine-edge-pool-review-nodes.adoc

> :_content-type: CONCEPT

Reviewer: ```suggestion
:_content-type: PROCEDURE
```

[id="machine-edge-pool-review-nodes_{context}"]

> = Check nodes created with edge compute pool

Reviewer: ```suggestion
= Verifying nodes that were created with edge compute pool
```

> After your cluster have been installed successfully, you can check the Machine created by Machine Set added on install time.

Reviewer: ```suggestion
After your install a cluster that uses AWS Local Zones, check the status of the machine that was created by the machine set that you added.
```

Author: What about this?:

> After your install a cluster that uses AWS Local Zones, check the status of the machine that was created by the machine set manifests created on install time.

Reviewer: ```suggestion
After your install a cluster that uses AWS Local Zones, check the status of the machine that was created by the machine set manifests created at install time.
```

> . Check the Machine Sets created from the subnet added to install-config.yaml:
> +
> [source,terminal]
> ----
> $ oc get machineset -n openshift-machine-api --show-labels
> NAME                                  DESIRED   CURRENT   READY   AVAILABLE   AGE    LABELS
> cluster-7xw5g-edge-us-east-1-nyc-1a   1         1         1       1           3h4m   machine.openshift.io/cluster-api-cluster=cluster-7xw5g
> cluster-7xw5g-worker-us-east-1a       1         1         1       1           3h4m   machine.openshift.io/cluster-api-cluster=cluster-7xw5g
> cluster-7xw5g-worker-us-east-1b       1         1         1       1           3h4m   machine.openshift.io/cluster-api-cluster=cluster-7xw5g
> cluster-7xw5g-worker-us-east-1c       1         1         1       1           3h4m   machine.openshift.io/cluster-api-cluster=cluster-7xw5g
> ----

Reviewer: Lowercase API object terms. Monospace actual API objects, e.g. `MachineSet`.

```suggestion
. To check the machine sets created from the subnet you added to the `install-config.yaml` file, enter the following command:
+
[source,terminal]
----
$ oc get machineset -n openshift-machine-api --show-labels
----
+
.Example output
[source,terminal]
----
NAME                                  DESIRED   CURRENT   READY   AVAILABLE   AGE    LABELS
cluster-7xw5g-edge-us-east-1-nyc-1a   1         1         1       1           3h4m   machine.openshift.io/cluster-api-cluster=cluster-7xw5g
cluster-7xw5g-worker-us-east-1a       1         1         1       1           3h4m   machine.openshift.io/cluster-api-cluster=cluster-7xw5g
cluster-7xw5g-worker-us-east-1b       1         1         1       1           3h4m   machine.openshift.io/cluster-api-cluster=cluster-7xw5g
cluster-7xw5g-worker-us-east-1c       1         1         1       1           3h4m   machine.openshift.io/cluster-api-cluster=cluster-7xw5g
----
```

. Check the Machines created from the Machine Sets:
+
[source,terminal]
----
$ oc get machines -n openshift-machine-api 
NAME                                        PHASE     TYPE          REGION      ZONE               AGE
cluster-7xw5g-edge-us-east-1-nyc-1a-wbclh   Running   c5d.2xlarge   us-east-1   us-east-1-nyc-1a   3h
cluster-7xw5g-master-0                      Running   m6i.xlarge    us-east-1   us-east-1a         3h4m
cluster-7xw5g-master-1                      Running   m6i.xlarge    us-east-1   us-east-1b         3h4m
cluster-7xw5g-master-2                      Running   m6i.xlarge    us-east-1   us-east-1c         3h4m
cluster-7xw5g-worker-us-east-1a-rtp45       Running   m6i.xlarge    us-east-1   us-east-1a         3h
cluster-7xw5g-worker-us-east-1b-glm7c       Running   m6i.xlarge    us-east-1   us-east-1b         3h
cluster-7xw5g-worker-us-east-1c-qfvz4       Running   m6i.xlarge    us-east-1   us-east-1c         3h

> ----

Reviewer: ```suggestion
. To check the machines that were created from the machine sets, enter the following command:
+
[source,terminal]
----
$ oc get machines -n openshift-machine-api 
----
+
.Example output
----
NAME                                        PHASE     TYPE          REGION      ZONE               AGE
cluster-7xw5g-edge-us-east-1-nyc-1a-wbclh   Running   c5d.2xlarge   us-east-1   us-east-1-nyc-1a   3h
cluster-7xw5g-master-0                      Running   m6i.xlarge    us-east-1   us-east-1a         3h4m
cluster-7xw5g-master-1                      Running   m6i.xlarge    us-east-1   us-east-1b         3h4m
cluster-7xw5g-master-2                      Running   m6i.xlarge    us-east-1   us-east-1c         3h4m
cluster-7xw5g-worker-us-east-1a-rtp45       Running   m6i.xlarge    us-east-1   us-east-1a         3h
cluster-7xw5g-worker-us-east-1b-glm7c       Running   m6i.xlarge    us-east-1   us-east-1b         3h
cluster-7xw5g-worker-us-east-1c-qfvz4       Running   m6i.xlarge    us-east-1   us-east-1c         3h
----
```

> . Check nodes with edge roles:
> +
> [source,terminal]
> ----
> $ oc get nodes -l node-role.kubernetes.io/edge
> NAME                           STATUS   ROLES         AGE    VERSION
> ip-10-0-207-188.ec2.internal   Ready    edge,worker   172m   v1.25.2+d2e245f
> ----

Reviewer: ```suggestion
. To check nodes with edge roles, enter the following command:
+
[source,terminal]
----
$ oc get nodes -l node-role.kubernetes.io/edge
----
+
.Example output
[source,terminal]
----
NAME                           STATUS   ROLES         AGE    VERSION
ip-10-0-207-188.ec2.internal   Ready    edge,worker   172m   v1.25.2+d2e245f
----
```
